src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/pages/MoveDetailsPage.kt

```diff
@@ -1,12 +1,27 @@
 package uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages
 
+import org.assertj.core.api.Assertions.assertThat
 import org.fluentlenium.core.annotation.PageUrl
+import uk.gov.justice.digital.hmpps.pecs.jpc.move.Move
+import java.time.format.DateTimeFormatter
 
 @PageUrl("http://localhost:8080/moves/{moveId}")
 class MoveDetailsPage : ApplicationPage() {
 
-  /**
-   * Moves are actually retrieved by their reference number but the URL contains the move ID.
-   */
-  fun isAtPageForMoveID(moveRefId: String) = this.isAt(moveRefId).let { this }
+  fun isAtPageFor(move: Move) {
+    this.isAt(move.moveId).let {
+      val source = this.pageSource()
+
+      assertThat(source).contains(move.person?.prisonNumber)
```

Comment:
Just asserting on page source for now.  Obviously we can be more rigorous but I think this is a reasonable start for now.